README.md

```diff
@@ -43,7 +48,9 @@ important items in the configuration file:
 
 2. Start the exporter by running the following command:
 ```
-python app.py
+hanadb_exporter -c config.json
```

Comment:
We don't need to do it right now, but we need to create also a Systemd unit file for the exporter. This way, it is easier to admin, and pacemaker can also manage this together with the other cluster resources.

Reply:
Perfect! This will come in a future PR